### PR TITLE
Change domain to opensourcepledge.com

### PR DESCRIPTION
**Pull request by vladh**
Things not changed:
* CNAME, leaving this to @chadwhitacre
* <https://cal.com/osspledge/ecosystem-call>
* <https://cal.com/osspledge/intro>

### CODE REVIEW

File: bin/commit-member-changes [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 if [[ $(git status --porcelain src/content/members) ]]; then
   git config user.name 'Open Source Pledge GitHub Action Runner'
-  git config user.email 'bot@osspledge.com'
+  git config user.email 'bot@opensourcepledge.com'
```
[the remark this answers is not in the document]
Comment (vladh) [reply bin/commit-member-changes:11]: Yep, this is just for the git logs, it doesn't do anything.

[the remark this answers is not in the document]
Comment (vladh) [reply bin/commit-member-changes]: I've changed this to `<>` for clarity.

**Review by chadwhitacre — APPROVED** [review]

File: src/pages/about.astro [hunk 2]
```diff
@@ -27,7 +27,12 @@ import Layout from "../layouts/Layout.astro";
 
         <h2>Governance and Structure</h2>
 
-        <p>Open Source Pledge is brought to you by <a href="https://sentry.io/welcome/">Sentry</a> and <a href="https://github.com/opensourcepledge/osspledge.com/graphs/contributors">contributors</a>.</p>
+        <p
+          >Open Source Pledge is brought to you by
+          <a href="https://sentry.io/welcome/">Sentry</a>
+          and
+          <a href="https://github.com/opensourcepledge/opensourcepledge.com/graphs/contributors">contributors</a>.</p
+        >
```
Comment (chadwhitacre) [inline src/pages/about.astro:35]: Didn't we get rid of this? Need a sync from `main`?

Comment (vladh) [reply src/pages/about.astro:35]: Nope — we got rid of it everywhere _but_ the about page.

**Pull request merged**